packages/@react-spectrum/dialog/src/DialogContainer.tsx

```diff
@@ -39,8 +39,9 @@ export function DialogContainer(props: SpectrumDialogContainerProps) {
 
   // React.Children.toArray mutates the children, and we need them to be stable
   // between renders so that the lastChild comparison works.
-  let child = null;
+  let child: ReactElement | null = null;
   if (Array.isArray(children)) {
+    // @ts-ignore
```

Comment:
could this be `ReactElement | undefined` instead of doing a `ts-ignore`?

Reply:
I swear i tried this and it failed. however, I tried it again and it works. So I've been able to remove it